src/main/java/com/frcteam3636/frc2024/subsystems/shooter/Shooter.kt

```diff
@@ -53,7 +69,14 @@ object Shooter : Subsystem {
                 .also { it.addRequirements(this) }
     }
 
+    fun runWithSetpoint(setpoint: Rotation2d): Command {
```

Comment:
Since this name could also refer to running the flywheels, I would name it like `startPivotingTo` instead. There should probably also be a `pivotTo` command that only ends once the setpoint has been reached, since that's often what we'll want anyway.